frontendGraphImpl/src/main/java/com/virtuslab/gitmachete/frontend/graph/impl/paint/GraphCellPainter.java

```diff
@@ -29,7 +29,12 @@
 
   @UIEffect
   protected int getRowHeight() {
-    return table.getRowHeight();
+    var font = table.getFont();
```

Comment:
Pls add a comment in the code:
1. when (semantically) the font is null and when not (if we know that)
2. that the purpose is to make graph scale alongside the default font from IntelliJ Settings